studio/components/interfaces/SQLEditor/SQLEditor.tsx

```diff
@@ -171,8 +173,33 @@ const SQLEditor = () => {
       // Refetching instead of invalidating since invalidate doesn't work with `enabled` flag
       refetchEntityDefinitions()
     },
-    onError(error) {
-      if (id) snap.addResultError(id, error)
+    onError(error: any) {
+      if (id) {
+        if (error.position && monacoRef.current) {
+          const editor = editorRef.current
+          const monaco = monacoRef.current
+
+          const formattedError = error.formattedError ?? ''
+          const lineError = formattedError.slice(formattedError.indexOf('LINE'))
+          const line = Number(lineError.slice(0, lineError.indexOf(':')).split(' ')[1])
+
+          const decorations = editor?.deltaDecorations(
+            [],
+            [
+              {
+                range: new monaco.Range(line, 1, line, 20),
+                options: { isWholeLine: true, inlineClassName: 'bg-amber-800' },
```

Comment:
`bg-amber-800` vs using red?

Reply:
i think amber is okay - its a warning, not something destructive
lets reserve using red for things that are destructive